src/config/index.js

```diff
@@ -19,7 +19,14 @@ const setConfig = update => {
   return writeJSONSync(configFilePath, mergeDeep(readJSONSync(configFilePath), update))
 }
 
+const isURLValid = () => {
```

Comment:
@barryadk This might be a candidate to use in that `validate` functionality we talked about earlier?

How you feel about where I've implemented this? Should it belong elsewhere?